spring-xd-deployment.tex

```diff
@@ -140,24 +140,11 @@ \subsection{Lambda architecture}
 \verb;stream create count --definition  ;\\*
 \verb;"tap:stream:ingest > extract-tags | jdbc";\\*
 
-\subsubsection {View layer in Spring XD}
+\subsubsection {Serving layer with Spring XD}
 
-Externalized via sinks, output of batch jobs. Microservice architecture
-Example of a web application reading data off a set of tables
-written on by Spring XD. 
-
-\subsubsection {Combining the two}
-
-Stream-driven batch jobs. Taps. 
-
-
-\subsubsection {Critique of the Lambda Architecture and Spring XD}
-
-An often raised objection to the LA is the duplication of work
-(Quote Jay Kreps) that occurs during processing. More specifically,
-the necessity of writing the same processing code twice: once for the
-speed and once for the batch layer. However, due to the nature of
-Spring XD, the same business code can be shared between the two 
-layers. This objection does not apply!
+Spring XD does not provide a serving layer of its own, but, once the 
+\emph{speed layer} and the \emph{batch layer} store their data in 
+\emph{views}, a serving layer can be easily built using Spring Boot in
+combination with Spring Data or Spring Data REST.
```

Comment:
Should we mention the REST endpoints exposed by the analytics metric repository being one such serving layer?